kolibri/content/constants/presets.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module contains constants representing the ids of FormatPreset.
+"""
+from django.utils.translation import ugettext_lazy as _
+
+# constants for FormatPreset
+HIGH_RES_VIDEO = "high_res_video"
+MID_RES_VIDEO = "mid_res_video"
+LOW_RES_VIDEO = "low_res_video"
+VECTOR_VIDEO = "vertor_video"
+
+EN_SUBTITLE = "en_subtitle"
+ES_SUBTITLE = "es_subtitle"
+ZH_SUBTITLE = "zh_subtitle"
```

Comment:
> In Kolibri, videos in different languages would typically live in different channels (or at least, under different content nodes), so the literal transcript would be the one in the same language as the video.

@jamalex That's sounds a lot like what we had in KA Lite and was the underlying cause for the issue [#3913](https://github.com/learningequality/ka-lite/issues/3913#issuecomment-130006565). Literal transcript must be unequivocally linked to **its own video** by more that just **being in the same language**.